src/main/java/hudson/plugins/ircbot/IrcNotifyStep.java

```diff
@@ -56,6 +59,10 @@
     private BuildToChatNotifier buildToChatNotifier = new DefaultBuildToChatNotifier();
     private MatrixJobMultiplier matrixNotifier = MatrixJobMultiplier.ONLY_PARENT;
 
+    // Instead of build status messages, send an arbitrary message to specified
```

Comment:
Note: As subsequent co-evolution of ircbot and instant-messaging plugins, maybe makes sense to evict this feature there, similarly to `extraMessage` done in @akomakom's branch.